plugins/threaded/include/lcms2_threaded.h

```diff
@@ -24,7 +24,7 @@
 
 #include "lcms2_plugin.h"
 
-#if !defined(CMS_IS_WINDOWS_) && defined(CMS_NO_PTHREADS)
+#ifdef CMS_NO_PTHREADS
```

Comment:
Reinstating `CMS_NO_PTHREADS` means that this plugin is effectively impossible to compile in Windows under MSVC.

https://github.com/mm2/Little-CMS/blob/c8ed7427b83df73f3039ef570c48c6b995c2a7ab/include/lcms2.h#L257-L263

`HasTHREADS` is only defined if pthreads was found.

Reply:
@mm2 if we keep my original change, the plugin builds and passes the test suite without issue under Windows. Was blocking MSVC compatibility your intent?

Reply:
Yep. That's for compiling the CMM with no thread support. Is refers NOT the phread library, but the capability of working with threads. This plugin needs threads so hence the check. Not sure if meson assumes this is the pthread library.

Reply:
Forgot to comment... this is a configuration toggle defined in lcms2.h
If HasTHREADS is defined then it is forced to 1, but otherwise is up to user to decide if the CMM should be thread aware or not.

Reply:
> Not sure if meson assumes this is the pthread library.

It's the Autotools script that assumes so:

https://github.com/mm2/Little-CMS/blob/486b5f537b2cbb7848590f2d153afccbffee4170/configure.ac#L240-L250

The author of the Meson script assumed the (p)threads library is always available:

https://github.com/mm2/Little-CMS/blob/486b5f537b2cbb7848590f2d153afccbffee4170/meson.build#L48-L51